File: frontend/src/services/ssoService.js
import apiClient from './api';

export const getSsoConfig = () => apiClient.get('/suite/sso/config');
export const updateSsoConfig = (dto) => apiClient.post('/suite/sso/config', dto);
export const getAuditLogs = () => apiClient.get('/suite/sso/audit');

File: backend/src/main/java/com/awais/hr/module/sso/service/SsoService.java
package com.awais.hr.module.sso.service;

import java.util.List;
import java.util.Map;

public interface SsoService {
    Map<String, Object> getSsoConfig();
    Map<String, Object> updateSsoConfig(Map<String, Object> body);
    List<Map<String, Object>> getAuditLogs();
}
